data/core/units/elves/Hero.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=Elvish Champion
     cost=30
     usage=fighter
-    description= _ "Elves, though not warlike by nature, are swift to learn; a relatively small amount of experience will turn an elf from a competent fighter into a master of combat. Those who are honored as heroes are strong with both sword and bow, and their skill once acquired does not fade for lack of practice."
+    description= _ "Elves that have grown more adept at swordsmanship often forego a shield in favor of a secondary shortsword, useful for both parrying enemy attacks and sneaking in extra strikes when engaging in melee combat. These diminutive blades are often known as 'Váthial', little heroes, for their third use as replacement swords for weaponless allies. More inexperienced fighters are easily disarmed by stronger opponents and quickly overwhelmed when only left with a bow to fight with. By offering their parrying swords to their more vulnerable comrades, elvish heroes grant them a second chance to fight or to retreat with their lives."
```

Comment:
s/shortsword/short sword/

I agree with CM about making up words. Also, you're describing game mechanics which do not exist. There is no case where a unit is rendered weaponless during combat, nor is there an ability to pass a weapon to such a unit. I like the idea, in general, but do not want to set expectations which are not met.

Reply:
"Shortsword" is also correct. If the in house style is short sword, I can change it. Lore does not always match up with gameplay perfectly, and I very much disagree with sacrificing depth in lore just because we don't have all the gameplay to describe everything that appears lore-wise.

Reply:
I'm all for adding depth in lore, but not at the expense of clarity over game mechanics. Can we find a different way to add lore here?

To Gregory's points I'll add that "sneak in an extra strike" also sounds like a game mechanic (it could be describing a unit that has an attack that's 8x4 when used offensively but 8x3 when used defensively).

Reply:
Point taken, if someone else is willing to rewrite those sentences, I will add them in my commit.

Reply:
```suggestion
    description= _ "Though not warlike by nature, some Elves actively seek combat, gaining fame and heroic stature for their combat prowess. Unlike many of their brethren, they have grown adept at swordsmanship; often forgoing a shield in favor of a secondary shortsword."
```

Reply:
Um, I just want to say that I approved this made-up word via IRC. My point about made-up words isn't that you shouldn't do it but that they should be designed to fit into the phonotactics of the Elvish language insofar as they are established by Elvish names.

Speaking of which, you definitely need a "po" note for the Elvish word.